tools/__tasks__/commercial/graph/output/standalone.commercial.ts.json

```diff
@@ -73,7 +162,7 @@
   "../projects/common/modules/commercial/commercial-features.ts"
  ],
  "../projects/commercial/modules/ad-verification/prepare-ad-verification.ts": [
-  "../../../../node_modules/@guardian/commercial-core/dist/cjs/index.d.ts",
+  "../../../../../commercial-core/dist/cjs/index.d.ts",
```

Comment:
Something weird seems to be going on with this file!

Reply:
Strange! I've dropped these commits for now